src/main/java/com/igot/cb/discussion/service/impl/DiscussionServiceImpl.java

```diff
@@ -1622,7 +1630,7 @@ private void fetchLikes(List<String> discussionIds, String userId, Map<String, B
                 .forEach(discussionId -> likesMap.put(discussionId, true));
     }
 
-    private void fetchBookmarks(List<String> discussionIds, String userId, String communityId, Map<String, Boolean> bookmarksMap) {
+    private void fetchBookmarks(List<String> discussionIds, String userId, List<String> communityId, Map<String, Boolean> bookmarksMap) {
```

Comment:
this is not good...
this method should be called in loop for each community...
this query may work... but this will search across all the partitions in Cassandra cluster which would decrease the performance